icu4c/source/i18n/simpletz.cpp

```diff
@@ -509,8 +509,10 @@ SimpleTimeZone::getOffset(uint8_t era, int32_t year, int32_t month, int32_t day,
 }
 
 void
-SimpleTimeZone::getOffsetFromLocal(UDate date, int32_t nonExistingTimeOpt, int32_t duplicatedTimeOpt,
-                                   int32_t& rawOffsetGMT, int32_t& savingsDST, UErrorCode& status) const {
+SimpleTimeZone::getOffsetFromLocal(
+    UDate date, UTimeZoneLocalOption nonExistingTimeOpt,
+    UTimeZoneLocalOption duplicatedTimeOpt,
+    int32_t& rawOffsetGMT, int32_t& savingsDST, UErrorCode& status) const {
```

Comment:
In C++ the { start in a new line on the left

Reply:
Why? It is very common to have the `{` at the end of the function signature.

Reply:
Anyway, I moved { to next line to match other methods in the same file.
I think it's a good idea to define style guide and automated formatter although.